site/en/install/docker.md

```diff
@@ -8,7 +8,7 @@ connect to the Internet, etc.). The
 [TensorFlow Docker images](https://hub.docker.com/r/tensorflow/tensorflow/){:.external}
 are tested for each release.
 
-Docker is the easiest way to enable TensorFlow [GPU support](./gpu.md) on Linux since only the
+Docker is the easiest way to enable TensorFlow [GPU support](https://www.tensorflow.org/install/pip) on Linux since only the
```

Comment:
`gpu.md` redirects to `https://www.tensorflow.org/install/pip`, so let's do the following:

```suggestion
Docker is the easiest way to enable TensorFlow [GPU support](./pip.md) on Linux since only the
```